### PR TITLE
Update widget-ci.yml

### PR DESCRIPTION
**Pull request by spau6**

### CODE REVIEW

**Review by amar-c8y — DISMISSED** [review]

File: .github/workflows/widget-ci.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
 
   pull_request:
     paths:
-      - cumulocity-asset-properties-widget/asset-properties-widget/**
+      - cumulocity-asset-properties-widget/**
```
Comment (ghost) [inline .github/workflows/widget-ci.yml]:
```suggestion
      - asset-properties-widget/**
```
use the correct folder path to look for changes. please refer documentation : https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

File: .github/workflows/widget-ci.yml
Comment (ghost) [inline .github/workflows/widget-ci.yml:1]: modify the path in skip file too. The paths in actual workflow and skip file must be the same.

**Review by ghost — CHANGES_REQUESTED** [review]

File: .github/workflows/widget-ci.yml
Comment (spau6) [reply .github/workflows/widget-ci.yml:1]: done

**Review by amar-c8y — APPROVED** [review]

**Comment by spau6** [comment]
HYG for successful run: https://github.com/SoftwareAG/cumulocity-asset-properties-widget/pull/10

**Review by ghost — APPROVED** [review]

**Pull request merged**